src/site/content/en/learn/accessibility/javascript/javascript.assess.yml

```
@@ -0,0 +1,23 @@
+setLeader: Test your knowledge of Javascript
+tabLabel: question
+questions:
+  - type: multiple-choice
+    cardinality: "1"
+    correctAnswers: "2"
```

```suggestion
    correctAnswers: "1"
```

@AaronForinton some more fun to be aware of. The options count starts with 0. So, because the correct answer is "Use JavaScript to toggle an element’s class, and add the style to your CSS stylesheet", it should be 1